lib/koans/19_streams.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Streams do
+  use Koans
+
+  @intro "Streams"
+
+  koan "Streams are lazier than Enum, they only return a struct" do
+    assert Stream.map([1, 2, 3], &(&1 + 1)).__struct__ == ___
+  end
+
+  koan "You have to ask it explicitly to do the operation" do
+    updated_list =
+      [1, 2, 3]
+      |> Stream.map(&(&1 + 1))
```

Comment:
The point of this example was more to show how to execute an operation on a Stream. In this very case, even if the use of a Stream isn't ideal it's still a possibility.

That's why I preferred to create another example that demonstrates that you can compute a selected part of the Stream.